docs/source/relayer.rst

```diff
@@ -35,14 +42,15 @@ To start relayer from command line, download UniversalLoginSDK from github and:
 
     yarn relayer:start
 
+.. _programmatically:
 
 Programmatically
 ^^^^^^^^^^^^^^^^
```

Comment:
To run relayer from your application you will need to create a relayer instance. Relayer constructor documentation below.